hazelcast/src/main/java/com/hazelcast/client/impl/ClusterViewListenerService.java

```diff
@@ -57,6 +60,12 @@
     private final boolean advancedNetworkConfigEnabled;
     private final AtomicBoolean pushScheduled = new AtomicBoolean();
     private final CoalescingDelayedTrigger delayedPartitionUpdateTrigger;
+    // This is an emulation of the pre-4.1 partition state version.
+    // We will increment this version if a partition table change is detected
+    // while sending partition table to the client.
+    private final AtomicInteger partitionTableVersion = new AtomicInteger();
```

Comment:
it can be better to document when this `partitionTableVersion` can be deleted from code.

Reply:
Most probably it will stay until the next major version of the client protocol. I'll add this.